### PR TITLE
Remove secrets from codebase

### PR DESCRIPTION
**Pull request by aluminiumgeek**
- remove secrets from codebase
- added a script that generates env secrets
- updated docs

### CODE REVIEW

File: .docker-compose-db.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ services:
     restart: always
     ports:
     - "5432:5432"
-    environment:
```
Comment (akariv) [inline .docker-compose-db.yaml:12]: These dbs should not be exposed externally, so pretty pointless to set up authentication here.

File: .docker-compose-db.yaml [hunk 2]
```diff
@@ -9,8 +9,8 @@ services:
     restart: always
     ports:
     - "5432:5432"
```
Comment (akariv) [inline .docker-compose-db.yaml]: This should be `expose` and not `ports` and not expose the port on the host.

File: .docker-compose-centralized.yaml [hunk 3]
```diff
@@ -6,14 +6,14 @@ services:
     - INSTANCE_ID
     - CKAN_K8S_SECRETS=/etc/ckan-conf/secrets/secrets-centralized.sh
     volumes:
-    - ./docker-compose/ckan-secrets-centralized.sh:/etc/ckan-conf/secrets/secrets-centralized.sh
```
Comment (akariv) [inline .docker-compose-centralized.yaml:9]: This file is to be used in local development only, so there's no 'harden' it...

**Comment by akariv** [comment]
Not sure I see the need in this PR.
None of the secrets here are actual secrets - DB connection strings are trivial exactly because they run inside the same docker deployment and are not exposed to the outside world.

Centralized configuration is even less important as it's used only for local development.

Overall it seems like an unnecessary complication...

**Review by akariv — CHANGES_REQUESTED** [review]

**Comment by aluminiumgeek** [comment]
> Not sure I see the need in this PR.

The PR is for first task in https://gitlab.com/datopian/tech/issues/62
If we don't really need it, then (sad) let's close it.
@pwalsh what do you think?

**Pull request merged**